packages/autorest.python/autorest/codegen/models/dictionary_type.py

```diff
@@ -129,3 +129,7 @@ def imports(self, **kwargs: Any) -> FileImport:
     @property
     def instance_check_template(self) -> str:
         return "isinstance({}, dict)"
+
+    @property
+    def type_description(self) -> str:
+        return f"{{{self.element_type.type_description}}}"
```

Comment:
we should make this a mapping between str: self.element_type.type_description. I'll update